pkg/shell/superuser.jsx

```diff
@@ -176,19 +176,19 @@ export class SuperuserDialogs extends React.Component {
 
     unlock(error) {
         this.superuser.Stop().always(() => {
-            if (this.superuser.Bridges.length == 1) {
-                this.start(this.superuser.Bridges[0], error);
-            } else {
-                this.set_unlock_state({
-                    method: this.superuser.Bridges[0],
-                    methods: this.superuser.Bridges,
-
-                    error: error,
-                    change: val => this.update_unlock_state({ method: val }),
-                    cancel: () => this.set_unlock_state({ closed: true }),
-                    apply: () => this.start(this.state.unlock_dialog_state.method)
-                });
-            }
+            cockpit.spawn(["sudo", "-v", "-n"], { err: "message" })
+                    .fail(e => {
+                        if (e.message && e.message.match("Sorry, user .+ may not run sudo on .+\\.")) {
```

Comment:
I'm a bit sceptical about pinning down human visible strings like this. It may slightly change over time, or even get translated. Perhaps run it with `LC_ALL=C` and just check for "may not run"?

Reply:
Discussed at `can_do_sudo`.